src/Failsafe.ts

```diff
@@ -18,7 +18,17 @@ export class Failsafe {
     ) {
     }
 
-    async run(scriptsName: string[]): Promise<ExitCode> {
+    async run(arguments_: string[]): Promise<ExitCode> {
+        const scriptArguments: {[script: string]: string[]} = {};
+        try {
+            this.parseArguments(arguments_, scriptArguments);
+        } catch (error: any) {
+            this.logger.error('failsafe', `${error}`);
+            return 1;
+        }
```

Comment:
Was not sure about the try-catch here… `parseArguments` could also return an optional error message instead of throwing an error.

Reply:
Yeah, I see what you mean; I tweaked it slightly.